src/drivers/boards/px4fmu-v2/board_config.h

```diff
@@ -215,6 +215,7 @@
 #define PX4_SPIDEV_EXT_ACCEL_MAG PX4_MK_SPI_SEL(PX4_SPI_BUS_EXT, 2)
 #define PX4_SPIDEV_EXT_BARO      PX4_MK_SPI_SEL(PX4_SPI_BUS_EXT, 3)
 #define PX4_SPIDEV_EXT_MPU       PX4_MK_SPI_SEL(PX4_SPI_BUS_EXT, 4)
```

Comment:
Using PX4_SPIDEV_EXT_MPU should be fine here. We should probably have a better generic external name.